client/question/components/TextWithMarkdownLinksSnippetDirectiveSpec.js

```diff
@@ -71,37 +73,42 @@ describe('TextWithMarkdownLinksSnippetDirective', function() {
       expect(template.html()).toContain(
         'abc <a href="https://www.google.com" target="_blank">def</a>');
 
-      parentScope.parentContent = 'abc [def](../docs/py-primer-light.html)';
+      parentScope.parentContent = 'abc [def](primer-url)';
       scope.$digest();
       expect(template.html()).toContain(
-        'abc <a href="../docs/py-primer-light.html" target="_blank">def</a>');
+        'abc <a href="..' + PRIMER_URL_DIRECTORY +
```

Comment:
Why not include the .. in PRIMER_URL_DIRECTORY?

(and/or make the name clearer to indicate whether it's the directory from the current working directory, or from TIE's basepath)

Reply:
makes sense. I'll just include the ..